jetty-osgi/test-jetty-osgi/pom.xml

```diff
@@ -394,18 +394,27 @@
       <version>5.0.1</version>
       <scope>test</scope>
     </dependency>
-    
+    <dependency>
+      <groupId>org.eclipse.jetty.http2</groupId>
+      <artifactId>http2-client</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty.http2</groupId>
+      <artifactId>http2-http-client-transport</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
   <build>
     <plugins>
+        <!-- downgrading to version 2.18 here as there's a known JVM crash issue with the 2.19.x series -->
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
-        <!-- downgrading to version 2.18 here as there's a known JVM crash issue with the 2.19.x series -->
-        <version>2.18.1</version>
+        <version>2.20.1</version>
```

Comment:
The version for the surefire plugin should be inherited from the main POM, otherwise it's going to remain behind when we update the main POM.

Reply:
Its not possible to upgrade the main pom to surefire 2.2.0.1. Or rather it would be, but then we would have to specifically back down the version in a number of subprojects - this is because this version of the surefire plugin hardcoded the addition of "--add-modules=java.se.ee", which provides only a few of the classes in javax.transaction package. As you can't have split packages in jdk9, then even if we put the javax.transaction-api jar on the classpath, it is ignored. See https://issues.apache.org/jira/browse/SUREFIRE-1403.

Reply:
Do the test pass with 2.20 ? If so, let's keep the main POM version.

Reply:
Yes, tests pass with same version as in main pom, so I changed the test-osgi/pom.xml to use the same version.